druid/src/widget/image.rs

```diff
@@ -113,17 +128,36 @@ impl ImageData {
             pixels: [].to_vec(),
             x_pixels: 0,
             y_pixels: 0,
+            format: ImageFormat::RgbaSeparate,
         }
     }
 
     /// Load an image from a DynamicImage from the image crate
     pub fn from_dynamic_image(image_data: image::DynamicImage) -> ImageData {
-        let rgb_image = image_data.to_rgba();
-        let sizeofimage = rgb_image.dimensions();
-        ImageData {
-            pixels: rgb_image.to_vec(),
-            x_pixels: sizeofimage.0,
-            y_pixels: sizeofimage.1,
+        match image_data.color() {
```

Comment:
this method is a bit cluttered but I don't have a great suggestion. I might add a constructor method to `ImageData`? But this isn't a blocker or anything.

Reply:
I have broken this up in to the match and then separate functions for with and without alpha.

This seems like a good idea as it means users can pick as well as having it workout what seems best.